### PR TITLE
Regression: Fix voip call wrap-up model not working

### PR DESCRIPTION
**Pull request by murtaza98**
## Issue(s)

image

### CODE REVIEW

**Review by KevLehman — DISMISSED** [review]

**Review by KevLehman — DISMISSED** [review]

File: apps/meteor/ee/app/livechat-enterprise/server/api/lib/tags.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { escapeRegExp } from '@rocket.chat/string-helpers';
-import LivechatTag from '@rocket.chat/models';
+import { LivechatTag } from '@rocket.chat/models';
```
Comment (ggazzo) [inline apps/meteor/ee/app/livechat-enterprise/server/api/lib/tags.js:2]: What do you think about convert to TS ? this way this is never going to happen again

Comment (murtaza98) [reply apps/meteor/ee/app/livechat-enterprise/server/api/lib/tags.js:2]: Yeah sounds good! Converted it to typescript. Please check

**Review by KevLehman — DISMISSED** [review]

**Comment by murtaza98** [comment]
Seems like some tests are breaking :see_no_evil: Let me check until then making it draft

**Comment by murtaza98** [comment]
> Seems like some tests are breaking see_no_evil Let me check until then making it draft

Fixed those build! This should be ready for review now

**Review by KevLehman — APPROVED** [review]

**Pull request merged**